docs/setup/cumulus-aws-template.yaml

```diff
@@ -198,6 +198,8 @@ Resources:
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/servicerequest"
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/covid_symptom__nlp_results"
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/covid_symptom__nlp_results_term_exists"
+              - !Sub "s3://${S3Bucket}/${EtlSubdir}/etl__completion"
+              - !Sub "s3://${S3Bucket}/${EtlSubdir}/etl__completion_encounters"
```

Comment:
Yes I removed these lines in the past to not confuse people - but that was because an empty crawler table was causing Library errors - I fixed that in https://github.com/smart-on-fhir/cumulus-library/pull/236 so these should be safe to add back in.